phpcs-compat.xml.dist

```diff
@@ -15,7 +15,7 @@
 	<exclude-pattern>*\.(?!php$)</exclude-pattern>
 
 	<!-- Configs -->
-	<config name="minimum_supported_wp_version" value="5.6" />
+	<config name="minimum_supported_wp_version" value="5.7" />
```

Comment:
WCPay 3.6 is released on Jan 19 while WP core 5.9 is released later on Jan 25. I think it would be better to update this minimum supported version for WP (here and other places) in WCPay 3.7?

Reply:
Fixed